usermods/usermod_v2_four_line_display_ALT/readme.md

```diff
@@ -1,43 +1,31 @@
 # I2C/SPI 4 Line Display Usermod ALT
 
 Thank you to the authors of the original version of these usermods. It would not have been possible without them!
-"usermod_v2_four_line_display"
-"usermod_v2_rotary_encoder_ui"
+"usermod_v2_four_line_display" (old but gold, removed since release 0.15.0)
+"usermod_v2_rotary_encoder_ui" (old but gold, removed since release 0.15.0)
 
-The core of these usermods are a copy of the originals. The main changes are to the FourLineDisplay usermod.
-The display usermod UI has been completely changed.
+The core of these usermods are a copy of the originals, the display usermod UI has been completely changed.
 
-
-The changes made to the RotaryEncoder usermod were made to support the new UI in the display usermod.
-Without the display, it functions identical to the original.
-The original "usermod_v2_auto_save" will not work with the display just yet.
+## Functionalities
 
 Press the encoder to cycle through the options:
 * Brightness
 * Speed
 * Intensity
 * Palette
 * Effect
-* Main Color (only if display is used)
-* Saturation (only if display is used)
+* Main Color
+* Saturation
 
```

Comment:
There is also a Presets in the cycle.

Reply:
Sorry I do not understand this comment, could you explain me?
Thanks.